test/spec_response.rb

```diff
@@ -78,7 +78,7 @@
     body.each { |part| parts << part }
 
     parts.must_equal ["foo", "bar", "baz"]
-    h['content-length'].must_equal '6'
+    h['content-length'].must_be_nil
```

Comment:
Fixed assertion for invalid behaviour (9 bytes in body, content length set to 6???)